safenode/src/network/event.rs

```diff
@@ -85,7 +85,9 @@ pub enum NetworkEvent {
         channel: MsgResponder,
     },
     /// Emitted when the DHT is updated
-    PeerAdded,
+    PeerAdded(PeerId),
+    /// Emitted when the DHT is updated
+    PeersAdded(Vec<PeerId>),
```

Comment:
PeerAdded can be PeersAdded with just one entry?

Reply:
Could be, absolutely.

Reply:
Fixed.